martor/templates/martor/guide.html

```diff
@@ -25,10 +24,6 @@
           </tr>
         {% endif %}
 
-        {% if mentions_enabled %}
```

Comment:
What did this used to do / why was it deleted?

Reply:
this inserts an empty row into the table, and i think it's necessary